web/views.py

```diff
@@ -333,4 +331,9 @@ def dispatch(self, request, path, *args, **kwargs):
             # requesting user doesn't belong to that child
             raise PermissionDenied()
 
+        request.META['HTTP_X-EXPERIMENT-CHILD-ID'] = path.split('/')[1]
```

Comment:
I need to delete that. I forgot for a minute that ember could actually look at the URL in the browser and was trying to find a way to pass the child id to the app, but it already had it.